build-tools/packages/build-cli/src/commands/collect/bundleStats.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import path from "path";
+import { execSync } from "child_process";
+import { existsSync, copySync, readJSONSync } from "fs-extra";
+import { Flags } from "@oclif/core";
+import { BaseCommand } from "../../base";
+
+export default class CollectBundlestats extends BaseCommand<typeof CollectBundlestats.flags> {
+    static description = `Find all bundle analysis artifacts and copy them into a central location to upload as build artifacts for later consumption`;
+    static flags = {
+        lernaOutput: Flags.string({
+            description: "Lerna Output",
+            default: `npx lerna list --all --json`,
+            required: false,
+            multiple: true,
+        }),
+        smallestAssetSize: Flags.integer({
+            description: `The smallest asset size in bytes that we deems to be correct. Adjust if we are testing for assets that are smaller.`,
+            default: 100,
+            required: false,
+        }),
+        ...BaseCommand.flags,
+    };
+
+    public async run(): Promise<void> {
+        const flags = this.processedFlags;
+        const lernaOutput = JSON.parse(execSync("npx lerna list --all --json").toString());
+
+        if (!Array.isArray(lernaOutput)) {
+            this.error("failed to get package information");
+        }
+
+        // Check each package location for a bundleAnalysis folder
+        // and copy it to a central location
+        let hasSmallAssetError = false;
+        const analysesDestPath = path.join(process.cwd(), "artifacts/bundleAnalysis");
+
+        // eslint-disable-next-line unicorn/no-array-for-each
```

Comment:
FWIW I think the lint warning is valid here. Switching to a `for...of` is more readable IMO.